src/fights.ts

```diff
@@ -361,20 +358,28 @@ function pygmyOptions(forceEquip: Item[] = []) {
 let _bestShadowRift: Location | null = null;
 export function bestShadowRift(): Location {
   if (!_bestShadowRift) {
-    _bestShadowRift =
-      chooseRift({
-        canAdventure: true,
-        sortBy: (l: Location) => {
-          const drops = getMonsters(l)
-            .map((m) => Object.keys(itemDrops(m)).map((s) => toItem(s)))
-            .flat();
-          return sum(drops, garboValue);
-        },
-      }) ?? $location.none;
+    _bestShadowRift = ClosedCircuitPayphone.chooseRift({
+      canAdventure: true,
+      sortBy: (l: Location) => {
+        setLocation(l);
+        // We probably aren't capping item drops with the penalty
+        // so we don't really need to compute the actual outfit (or the dropModifier for that matter actually)
+        const dropModifier = 1 + numericModifier("Item Drop") / 100;
+        return sum(getMonsters(l), (m) => {
+          const monsterDrops = itemDrops(m);
+          const items = Object.keys(monsterDrops).map(toItem);
+          const rates = Object.values(monsterDrops);
```

Comment:
I'm assuming you did this because Object.entries doesn't return an array, it returns an IterableIterator--we can use either `Array.from` or spread syntax to convert that to an array. so `[...Object.entries(monsterDrops)]` would have worked there. Again, moot because I think below handles it better